components/eamxx/docs/user/nudging.md

```diff
@@ -0,0 +1,25 @@
+# Nudging in EAMxx
+
+Nudging is supported in EAMxx.
```

Comment:
Some things we might want to add (in this PR or another):
1. to nudge to ERA5 data, we need to be able to dictionary between the EAMxx name and the ERA5 name for each variable, right? Would be worth mentioning how to do that. 
2. the fact that we can only nudge coarser grid data makes me think that regridding is done using an internally calculated bilinear remap rather than by passing the mapping file you want the code to use like we've done everywhere else. Otherwise nudging to finer data would just require passing conservative mapping weights. Sorry for being out of the loop, but why are we handling this case differently?
3. it's fine if Naser doesn't include info on weighted nudging here, but we definitely need it. We also need to include the specific commands needed to do remap nudging.
4. We should mention how data to nudge to can be split across many files. 
5. I kinda like how instead of long descriptions of what each command does, there's just an example with all of the commands you'd need to tinker with to get things working. But I guess we need to supply a bit of extra info like what the units of the nudging timescale are, whether nudging field names need to be eamxx format or reanalysis format, what the source_pressure_type is about, etc.

Reply:
i'm definitely not good at fully thinking through a wide range of user needs ahead of time.. you guys have very cool suggestions
 
just two questions - i thought the weighted nudging && nudging-to-coarse data simultaneously has been addressed since Luca's PR (the one changed the order of horizontal and vertical mapping)? as long as the weight file is on the model grid (horz+vert)

another thing is Aaron's PR to change the startdate in TimeInterpolation (https://github.com/E3SM-Project/scream/pull/2635) - it's very helpful for my own tests, just curious about its progress

Reply:
oh i see, thanks! i feel this check may can be removed after #2701 (it seems like the interpolation order caused the previous failures since the previous perform_vertical_interpolation needs to be on the same horiz grid) 

> We still have a runtime check to forbid this: https://github.com/E3SM-Project/scream/blob/305bde73db9d3c641e8808caa8371ad399256d9d/components/eamxx/src/physics/nudging/eamxx_nudging_process_interface.cpp#L138C1-L142C60